examples/cfd/external_aerodynamics/domino_sensitivity/main.py

```diff
@@ -0,0 +1,491 @@
+                - 'geometry_coordinates': Array of geometry point coordinates
+                - 'geometry_normal_sensitivity': Array of sensitivity values for each point
+                - 'pred_surf_pressure': Array of predicted surface pressure values [Pa]
+                - 'pred_surf_wall_shear_stress': Array of predicted wall shear stress values [τx, τy, τz] [Pa]
+                - 'aerodynamic_force': Array of total computed aerodynamic force [Fx, Fy, Fz] [N]
+
+        Example:
+            >>> import pyvista as pv
+            >>> from domino_sensitivity import DoMINOInference
+            >>>
+            >>> # Load geometry
+            >>> mesh = pv.read("car.stl")
+            >>>
+            >>> # Initialize inference
+            >>> domino = DoMINOInference(cfg)
+            >>>
+            >>> # Run inference
+            >>> results = domino(
+            ...     mesh=mesh,
+            ...     stream_velocity=30.0,
+            ...     stencil_size=7,
+            ...     air_density=1.205
+            ... )
+            >>>
+            >>> # Access results
+            >>> forces = results['aerodynamic_force']
+            >>> print(f"Drag force: {forces[0]:.2f} N")
+        """
+
+        datapipe = DesignDatapipe(
+            mesh=mesh,
+            bounding_box=self.bounding_box_min_max,
+            bounding_box_surface=self.bounding_box_surface_min_max,
+            grid_resolution=self.cfg.model.interp_res,
+            stencil_size=stencil_size,
+        )
+        dataloader = torch.utils.data.DataLoader(
+            datapipe, batch_size=2**13, shuffle=False
+        )
+
+        input_dict = {
+            k: torch.from_numpy(np.expand_dims(np.float32(v), axis=0)).to(self.device)
+            for k, v in datapipe.out_dict.items()
+        }
+        input_dict["stream_velocity"] = torch.tensor(
+            stream_velocity, dtype=torch.float32, device=self.device
+        )
+        input_dict["air_density"] = torch.tensor(
+            air_density, dtype=torch.float32, device=self.device
+        )
+
+        surface_keys: list[str] = [
+            "surface_mesh_centers",
+            "surface_mesh_neighbors",
+            "surface_normals",
+            "surface_neighbors_normals",
+            "surface_areas",
+            "surface_neighbors_areas",
+            "pos_surface_center_of_mass",
+        ]
+
+        # aerodynamic_force = torch.zeros(3, dtype=torch.float32, device=self.device)
+        aerodynamic_force = np.zeros(3, dtype=np.float32)
+        pred_surf_batches: list[np.ndarray] = []
+        geometry_coordinates = (
+            input_dict["geometry_coordinates"].detach().cpu().numpy()[0]
+        )
+        geometry_sensitivity: np.ndarray = np.zeros_like(geometry_coordinates)
+
+        def _print_memory_usage(label: str) -> None:
+            return
+            print(
+                f"VRAM usage after {label:<20}: {(torch.cuda.memory_allocated()/(1024**3)):.2f} GB"
+            )
```

Comment:
Dead code?

Reply:
Good catch, sorry! Yeah, I was using this for debugging. I think I'll refactor the dataclass to accept a `verbose` boolean flag to guard this (and other prints) behind.

Reply:
Fixed: 7d0eecdd8fcc98125a73db0d31da8a05468ad7cd